model/pdata/common.go

```diff
@@ -363,6 +363,37 @@ func (a AttributeValue) Equal(av AttributeValue) bool {
 	return false
 }
 
+// String returns the string representation of the AttributeValue.
+func (a AttributeValue) String() string {
```

Comment:
Worried about calling this "String" and the difference between this and "StringVal". What do you think? Was thinking "AsString" signals better and matches the pattern for "AsMap", "AsTime".

We can also make the "String()" to return "AsString" for the purpose of debug logging?

Reply:
Yeah I wondered about that as well. Happy to go w/ AsString, not sure if we need to add a String method as well... that starts getting confusing :D

Reply:
Ok, let's do only AsString for the moment and add a comment about the difference compare with StringVal

Reply:
Fixed.